scripts/check_changeset.js

```diff
@@ -34,22 +45,14 @@ async function checkVersion() {
 }
 
 async function checkBump() {
-	const result = spawnSync("pnpm", ["changeset", "version"], {
-		stdio: "pipe"
-	});
-	if (result.status !== 0) {
-		console.error("Check changeset bump failed", result.stderr.toString());
-		process.exit(1);
-	} else {
+	try {
+		await $`pnpm changeset version`;
```

Comment:
is this the same as previous one,  the previous one include two scenario throw exception and status !== 0 but the current one only exit when throw exception ? does the $`pnpm changeset version` have return value ?

Reply:
Good catch. Fix this at https://github.com/web-infra-dev/rspack/pull/2880